di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -0,0 +1,109 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Transform: AWS::Serverless-2016-10-31
+Globals:
+  Function:
+    Timeout: 30
+
+Description: >-
+  This creates the lambda functions to stub CIMIT lambdas for getContraIndicators, getContraIndicatorCredential, 
+  putContraIndicators and postMitigations.
+
+Parameters:
+  VpcStackName:
+    Description: >
+      The name of the stack that defines the VPC in which this container will
+      run.
+    Type: String
+  Environment:
+    Description: The name of the environment to deploy to.
+    Type: String
+    AllowedPattern: ((production)|(build)|(dev.*))
+
+Resources:
+  # lambda to replicate cimit getContraIndicators
+  GetContraIndicatorsLambda:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_109: this requires a broad set of permissions
+    DependsOn:
+      - "GetContraIndicatorsFunctionLogGroup"
+    Properties:
+      FunctionName: !Sub "getContraIndicators-${Environment}"
+      CodeUri: "../lambdas/get-contra-indicators"
+      Handler: uk.gov.di.ipv.core.getcontraindicators.GetContraIndicatorsHandler::handleRequest
+      Runtime: java17
+      PackageType: Zip
+      Architectures:
+        - arm64
+      Environment:
+        Variables:
+          ENVIRONMENT: !Sub "${Environment}"
+      VpcConfig:
+        SubnetIds:
+          - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdA
+          - Fn::ImportValue: !Sub ${VpcStackName}-ProtectedSubnetIdB
+        SecurityGroupIds:
+          - !GetAtt CimitLambdaSecurityGroup.GroupId
+      Policies:
+        - VPCAccessPolicy: { }
+        - Statement:
+            - Sid: AllowGetSSMParam
+              Effect: Allow
+              Action:
+                - "ssm:GetParameter"
+              Resource:
+                - !Sub "arn:aws:ssm:eu-west-2:${AWS::AccountId}:parameter/stubs/*"
+
+  GetContraIndicatorsFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 14
+      LogGroupName: !Sub "/aws/lambda/get-contra-indicators-${Environment}"
```

Comment:
The log group name needs to be named as per the lambda, so
`!Sub "/aws/lambda/getContraIndicators-${Environment}"`
Currently we're getting an empty log group created by this template and a log group named per the lambda that gets autocreated